src/cli/commands/run.js

```diff
@@ -30,8 +30,17 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
     if (!visitedBinFolders.has(binFolder)) {
       if (await fs.exists(binFolder)) {
         for (const name of await fs.readdir(binFolder)) {
-          binCommands.push(name);
-          scripts[name] = quoteForShell(path.join(binFolder, name));
+          //consider only .cmd scripts on Windows and show them without the extension - #624
+          if (process.platform === 'win32') {
+            if (name.indexOf('.cmd') !== -1) {
+              const strippedName = name.substring(0, name.indexOf('.cmd'));
```

Comment:
Same potential issue as the above comment. Should check that ".cmd" is the end of the string.